EssentialFeed/EssentialFeedTests/Feed/API/FeedImageDataMapperTests.swift

```diff
@@ -6,7 +6,7 @@ import XCTest
 import EssentialFeed
 
 class FeedImageDataMapperTests: XCTestCase {
-	func test_map_throwsErrorOnNon200HTTPResponse() throws {
+	func test_map_throwsErrorOnNon2xxHTTPResponse() throws {
```

Comment:
`FeedImageDataMapper` has a slightly different requirements and it should fail if response is not 200. It should not accept 201...299 (you can find 201 in the `samples`)
Please revert this change
```suggestion
	func test_map_throwsErrorOnNon200HTTPResponse() throws {
```

Reply:
done